src/main/java/com/team2502/robot2017/subsystem/DriveTrainSubsystem.java

```diff
@@ -86,61 +86,39 @@ else if(xLevel < 0)
      * @param isLeftSide Whether or not it is the left joystick/side
      * @return the speed of the robot
      */
-    private double getSpeed(boolean isLeftSide)
+    private Pair<Double, Double> getSpeed()
     {
         double joystickLevel = 0.0D;
         // Get the base speed of the robot
-        joystickLevel = isLeftSide ? -OI.JOYSTICK_DRIVE_LEFT.getY() : -OI.JOYSTICK_DRIVE_RIGHT.getY();
+        joystickLevel = -OI.JOYSTICK_DRIVE_LEFT.getY();
 
         // Only increase the speed by a small amount
-        if(isLeftSide)
-        {
-            double diff = joystickLevel - lastLeft;
-            if(diff > 0.1D) { joystickLevel = lastLeft + 0.1D; }
-            else if(diff < 0.1D) { joystickLevel = lastLeft - 0.1D; }
-            lastLeft = joystickLevel;
-        }
-        else
-        {
-            double diff = joystickLevel - lastRight;
-            if(diff > 0.1D) { joystickLevel = lastRight + 0.1D; }
-            else if(diff < 0.1D) { joystickLevel = lastRight - 0.1D; }
-            lastRight = joystickLevel;
-        }
+        double diff = joystickLevel - lastLeft;
+        if(diff > 0.1D) { joystickLevel = lastLeft + 0.1D; }
+        else if(diff < 0.1D) { joystickLevel = lastLeft - 0.1D; }
```

Comment:
Again, a general comment: rather than raw values like 0.1D, giving the value a const (static final in java?) name and using the name instead 1) increases readability (i.e., wt* is 0.1D??? ;) ), 2) reduces code change errors when you decide to adjust the value to 0.2D and change 9 out of the 10 uses in the file :).

(this isn't even really part of this diff, of course. Like I said, general things to keep in mind)